automation/infrastructure/terraform/kcs/install_kubeflow/monitoring.tf

```diff
@@ -0,0 +1,46 @@
+resource "null_resource" "install_monitoring" {
+  depends_on = [null_resource.now,
+    null_resource.helm_setup,
+    null_resource.prepare_prometheus,
+    null_resource.install_prometheus]
+}
+
+resource "null_resource" "helm_setup" {
+  count = var.kubeflow_monitoring == "prometheus" ? 1 : 0
+
+  provisioner "local-exec" {
+    command = <<EOT
+    helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+    helm repo update
+    EOT
+  }
+}
+
+resource "null_resource" "prepare_prometheus" {
```

Comment:
lets name this create_monitoring_namespace and either use user provided name or default to kubeflow_monitoring. that way its more flexible and user can choose to install in his own namespace if needed. also we can even have a variable or check if namespace exists and create if does not.

Reply:
Good idea, fixing

Reply:
Fixed